### ISSUE TITLE
【EESS】New submissions for Wed, 13 Dec 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#983]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
### Impact of Array Configuration on Head-Mounted Display Performance at  mmWave Bands
 - **Authors:** Alexander Marinšek, Xuesong Cai, Lieven De Strycker, Fredrik Tufvesson, Liesbet Van der Perre
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2312.07356
 - **Pdf link:** https://arxiv.org/pdf/2312.07356
 - **Abstract**
 Immersing a user in life-like extended reality (XR) scenery using a head-mounted display (HMD) with a constrained form factor and hardware complexity requires remote rendering on a nearby edge server or computer. Millimeter-wave (mmWave) communication technology can provide sufficient data rate for wireless XR content transmission. However, mmWave channels exhibit severe sparsity in the angular domain. This means that distributed antenna arrays are required to cover a larger angular area and to combat outage during HMD rotation. At the same time, one would prefer fewer antenna elements/arrays for a lower complexity system. Therefore, it is important to evaluate the trade-off between the number of antenna arrays and the achievable performance to find a proper practical solution. This work presents indoor 28 GHz mmWave channel measurement data, collected during HMD mobility, and studies the dominant eigenmode (DE) gain. DE gain is a significant factor in understanding system performance since mmWave channel sparsity and eigenmode imbalance often results in provisioning the majority of the available power to the DE. Moreover, it provides the upper performance bounds for widely-adopted analog beamformers. We propose 3 performance metrics - gain trade-off, gain volatility, and minimum service trade-off - for evaluating the performance of a multi-array HMD and apply the metrics to indoor 28 GHz channel measurement data. Evaluation results indicate, that 3 arrays provide stable temporal channel gain. Adding a 4th array further increases channel capacity, while any additional arrays do not significantly increase physical layer performance.
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### On the notion of Hallucinations from the lens of Bias and Validity in  Synthetic CXR Images
 - **Authors:** Gauri Bhardwaj, Yuvaraj Govindarajulu, Sundaraparipurnan Narayanan, Pavan Kulkarni, Manojkumar Parmar
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2312.06979
 - **Pdf link:** https://arxiv.org/pdf/2312.06979
 - **Abstract**
 Medical imaging has revolutionized disease diagnosis, yet the potential is hampered by limited access to diverse and privacy-conscious datasets. Open-source medical datasets, while valuable, suffer from data quality and clinical information disparities. Generative models, such as diffusion models, aim to mitigate these challenges. At Stanford, researchers explored the utility of a fine-tuned Stable Diffusion model (RoentGen) for medical imaging data augmentation. Our work examines specific considerations to expand the Stanford research question, Could Stable Diffusion Solve a Gap in Medical Imaging Data? from the lens of bias and validity of the generated outcomes. We leveraged RoentGen to produce synthetic Chest-XRay (CXR) images and conducted assessments on bias, validity, and hallucinations. Diagnostic accuracy was evaluated by a disease classifier, while a COVID classifier uncovered latent hallucinations. The bias analysis unveiled disparities in classification performance among various subgroups, with a pronounced impact on the Female Hispanic subgroup. Furthermore, incorporating race and gender into input prompts exacerbated fairness issues in the generated images. The quality of synthetic images exhibited variability, particularly in certain disease classes, where there was more significant uncertainty compared to the original images. Additionally, we observed latent hallucinations, with approximately 42% of the images incorrectly indicating COVID, hinting at the presence of hallucinatory elements. These identifications provide new research directions towards interpretability of synthetic CXR images, for further understanding of associated risks and patient safety in medical applications.
### MS-Twins: Multi-Scale Deep Self-Attention Networks for Medical Image  Segmentation
 - **Authors:** Jing Xu
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2312.07128
 - **Pdf link:** https://arxiv.org/pdf/2312.07128
 - **Abstract**
 Although transformer is preferred in natural language processing, few studies have applied it in the field of medical imaging. For its long-term dependency, the transformer is expected to contribute to unconventional convolution neural net conquer their inherent spatial induction bias. The lately suggested transformer-based partition method only uses the transformer as an auxiliary module to help encode the global context into a convolutional representation. There is hardly any study about how to optimum bond self-attention (the kernel of transformers) with convolution. To solve the problem, the article proposes MS-Twins (Multi-Scale Twins), which is a powerful segmentation model on account of the bond of self-attention and convolution. MS-Twins can better capture semantic and fine-grained information by combining different scales and cascading features. Compared with the existing network structure, MS-Twins has made significant progress on the previous method based on the transformer of two in common use data sets, Synapse and ACDC. In particular, the performance of MS-Twins on Synapse is 8% higher than SwinUNet. Even compared with nnUNet, the best entirely convoluted medical image segmentation network, the performance of MS-Twins on Synapse and ACDC still has a bit advantage.
### Empirical Validation of Conformal Prediction for Trustworthy Skin  Lesions Classification
 - **Authors:** Jamil Fayyad, Shadi Alijani, Homayoun Najjaran
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2312.07460
 - **Pdf link:** https://arxiv.org/pdf/2312.07460
 - **Abstract**
 Uncertainty quantification is a pivotal field that contributes to the realization of reliable and robust systems. By providing complementary information, it becomes instrumental in fortifying safe decisions, particularly within high-risk applications. Nevertheless, a comprehensive understanding of the advantages and limitations inherent in various methods within the medical imaging field necessitates further research coupled with in-depth analysis. In this paper, we explore Conformal Prediction, an emerging distribution-free uncertainty quantification technique, along with Monte Carlo Dropout and Evidential Deep Learning methods. Our comprehensive experiments provide a comparative performance analysis for skin lesion classification tasks across the three quantification methods. Furthermore, We present insights into the effectiveness of each method in handling Out-of-Distribution samples from domain-shifted datasets. Based on our experimental findings, our conclusion highlights the robustness and consistent performance of conformal prediction across diverse conditions. This positions it as the preferred choice for decision-making in safety-critical applications.
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
### Impact of Array Configuration on Head-Mounted Display Performance at  mmWave Bands
 - **Authors:** Alexander Marinšek, Xuesong Cai, Lieven De Strycker, Fredrik Tufvesson, Liesbet Van der Perre
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2312.07356
 - **Pdf link:** https://arxiv.org/pdf/2312.07356
 - **Abstract**
 Immersing a user in life-like extended reality (XR) scenery using a head-mounted display (HMD) with a constrained form factor and hardware complexity requires remote rendering on a nearby edge server or computer. Millimeter-wave (mmWave) communication technology can provide sufficient data rate for wireless XR content transmission. However, mmWave channels exhibit severe sparsity in the angular domain. This means that distributed antenna arrays are required to cover a larger angular area and to combat outage during HMD rotation. At the same time, one would prefer fewer antenna elements/arrays for a lower complexity system. Therefore, it is important to evaluate the trade-off between the number of antenna arrays and the achievable performance to find a proper practical solution. This work presents indoor 28 GHz mmWave channel measurement data, collected during HMD mobility, and studies the dominant eigenmode (DE) gain. DE gain is a significant factor in understanding system performance since mmWave channel sparsity and eigenmode imbalance often results in provisioning the majority of the available power to the DE. Moreover, it provides the upper performance bounds for widely-adopted analog beamformers. We propose 3 performance metrics - gain trade-off, gain volatility, and minimum service trade-off - for evaluating the performance of a multi-array HMD and apply the metrics to indoor 28 GHz channel measurement data. Evaluation results indicate, that 3 arrays provide stable temporal channel gain. Adding a 4th array further increases channel capacity, while any additional arrays do not significantly increase physical layer performance.
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
### Large Foundation Models for Power Systems
 - **Authors:** Chenghao Huang, Siyang Li, Ruohong Liu, Hao Wang, Yize Chen
 - **Subjects:** Systems and Control (eess.SY); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2312.07044
 - **Pdf link:** https://arxiv.org/pdf/2312.07044
 - **Abstract**
 Foundation models, such as Large Language Models (LLMs), can respond to a wide range of format-free queries without any task-specific data collection or model training, creating various research and application opportunities for the modeling and operation of large-scale power systems. In this paper, we outline how such large foundation model such as GPT-4 are developed, and discuss how they can be leveraged in challenging power and energy system tasks. We first investigate the potential of existing foundation models by validating their performance on four representative tasks across power system domains, including the optimal power flow (OPF), electric vehicle (EV) scheduling, knowledge retrieval for power engineering technical reports, and situation awareness. Our results indicate strong capabilities of such foundation models on boosting the efficiency and reliability of power system operational pipelines. We also provide suggestions and projections on future deployment of foundation models in power system applications.
## Keyword: video retrieval
There is no result 
## Keyword: mobile
There is no result 
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### CSI-Based Cross-Domain Activity Recognition via Zero-Shot Prototypical  Networks
 - **Authors:** Guillermo Diaz, Iker Sobron, Inaki Eizmendi, Iratxe Landa, Manuel Velez
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2312.07076
 - **Pdf link:** https://arxiv.org/pdf/2312.07076
 - **Abstract**
 The cross-domain capability of wireless sensing is currently one of the major challenges on human activity recognition (HAR) based on the channel state information (CSI) of wireless signals. The difficulty of labeling samples from new domains has encouraged the use of few and zero shot strategies. In this context, prototype networks have attracted attention due to their reasonable cross-domain transferability. This paper presents a novel zero-shot prototype recurrent convolutional network that implements a zero-shot learning strategy for HAR via CSI. This method extracts the prototypes from an available source domain to classify unseen and unlabeled data from the target domain for the same or similar classes. The experiments have been developed using three datasets with real measurements, and the results include an inter-datasets evaluation. Overall, the results improve the state of the art and make it a promising solution for cross-domain HAR.
### Plug-and-play imaging with model uncertainty quantification in radio  astronomy
 - **Authors:** Matthieu Terris, Chao Tang, Adrian Jackson, Yves Wiaux
 - **Subjects:** Image and Video Processing (eess.IV); Instrumentation and Methods for Astrophysics (astro-ph.IM)
 - **Arxiv link:** https://arxiv.org/abs/2312.07137
 - **Pdf link:** https://arxiv.org/pdf/2312.07137
 - **Abstract**
 Plug-and-Play (PnP) algorithms are appealing alternatives to proximal algorithms when solving inverse imaging problems. By learning a Deep Neural Network (DNN) behaving as a proximal operator, one waives the computational complexity of optimisation algorithms induced by sophisticated image priors, and the sub-optimality of handcrafted priors compared to DNNs. At the same time, these methods inherit from the versatility of optimisation algorithms allowing to minimise a large class of objective functions. Such features are highly desirable in radio-interferometric (RI) imaging in astronomy, where the data size, the ill-posedness of the problem and the dynamic range of the target reconstruction are critical. In a previous work, we introduced a class of convergent PnP algorithms, dubbed AIRI, relying on a forward-backward backbone, with a differentiable data-fidelity term and dynamic range-specific denoisers trained on highly pre-processed unrelated optical astronomy images. Here, we show that AIRI algorithms can benefit from a constrained data fidelity term at the mere cost of transferring to a primal-dual forward-backward algorithmic backbone. Moreover, we show that AIRI algorithms are robust to strong variations in the nature of the training dataset: denoisers trained on MRI images yield similar reconstructions to those trained on astronomical data. We additionally quantify the model uncertainty introduced by the randomness in the training process and suggest that AIRI algorithms are robust to model uncertainty. Eventually, we propose an exhaustive comparison with methods from the radio-astronomical imaging literature and show the superiority of the proposed method over the state-of-the-art.
### Combined Invariant Subspace \& Frequency-Domain Subspace Method for  Identification of Discrete-Time MIMO Linear Systems
 - **Authors:** Jingze You, Chao Huang, Hao Zhang
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2312.07298
 - **Pdf link:** https://arxiv.org/pdf/2312.07298
 - **Abstract**
 Recently, a novel system identification method based on invariant subspace theory is introduced, aiming to address the identification problem of continuous-time (CT) linear time-invariant (LTI) systems by combining time-domain and frequency-domain methods. Subsequently, the combined Invariant-Subspace and Subspace Identification Method (cISSIM) is introduced, enabling direct estimation of CT LTI systems in state-space forms. It produces consistent estimation that is robust in an error-in-variable and slow-sampling conditions, while no pre-filtering operation of the input-output signals is needed. This paper presents the discrete-cISSIM, which extends cISSIM to discrete-time (DT) systems and offers the following improvements: 1) the capability to utilize arbitrary discrete periodic excitations while cISSIM uses multi-sine signals; 2) a faster estimation with reduced computational complexity is proposed; 3) the covariance estimation problem can be addressed concurrently with the system parameter estimation. An implementation of discrete-cISSIM by MATLAB has also been provided.
### Impact of Array Configuration on Head-Mounted Display Performance at  mmWave Bands
 - **Authors:** Alexander Marinšek, Xuesong Cai, Lieven De Strycker, Fredrik Tufvesson, Liesbet Van der Perre
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2312.07356
 - **Pdf link:** https://arxiv.org/pdf/2312.07356
 - **Abstract**
 Immersing a user in life-like extended reality (XR) scenery using a head-mounted display (HMD) with a constrained form factor and hardware complexity requires remote rendering on a nearby edge server or computer. Millimeter-wave (mmWave) communication technology can provide sufficient data rate for wireless XR content transmission. However, mmWave channels exhibit severe sparsity in the angular domain. This means that distributed antenna arrays are required to cover a larger angular area and to combat outage during HMD rotation. At the same time, one would prefer fewer antenna elements/arrays for a lower complexity system. Therefore, it is important to evaluate the trade-off between the number of antenna arrays and the achievable performance to find a proper practical solution. This work presents indoor 28 GHz mmWave channel measurement data, collected during HMD mobility, and studies the dominant eigenmode (DE) gain. DE gain is a significant factor in understanding system performance since mmWave channel sparsity and eigenmode imbalance often results in provisioning the majority of the available power to the DE. Moreover, it provides the upper performance bounds for widely-adopted analog beamformers. We propose 3 performance metrics - gain trade-off, gain volatility, and minimum service trade-off - for evaluating the performance of a multi-array HMD and apply the metrics to indoor 28 GHz channel measurement data. Evaluation results indicate, that 3 arrays provide stable temporal channel gain. Adding a 4th array further increases channel capacity, while any additional arrays do not significantly increase physical layer performance.
### Empirical Validation of Conformal Prediction for Trustworthy Skin  Lesions Classification
 - **Authors:** Jamil Fayyad, Shadi Alijani, Homayoun Najjaran
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2312.07460
 - **Pdf link:** https://arxiv.org/pdf/2312.07460
 - **Abstract**
 Uncertainty quantification is a pivotal field that contributes to the realization of reliable and robust systems. By providing complementary information, it becomes instrumental in fortifying safe decisions, particularly within high-risk applications. Nevertheless, a comprehensive understanding of the advantages and limitations inherent in various methods within the medical imaging field necessitates further research coupled with in-depth analysis. In this paper, we explore Conformal Prediction, an emerging distribution-free uncertainty quantification technique, along with Monte Carlo Dropout and Evidential Deep Learning methods. Our comprehensive experiments provide a comparative performance analysis for skin lesion classification tasks across the three quantification methods. Furthermore, We present insights into the effectiveness of each method in handling Out-of-Distribution samples from domain-shifted datasets. Based on our experimental findings, our conclusion highlights the robustness and consistent performance of conformal prediction across diverse conditions. This positions it as the preferred choice for decision-making in safety-critical applications.
